com.unity.multiplayer.mlapi/Tests/Runtime/com.unity.multiplayer.mlapi.runtimetests.asmdef

```diff
@@ -1,11 +1,22 @@
 {
     "name": "Unity.Multiplayer.MLAPI.RuntimeTests",
+    "rootNamespace": "MLAPI.RuntimeTests",
```

Comment:
I just changed this the rest was Unity's work but this looks more correct now overall? Looks like this asmdef was created by hand and never touched?

Reply:
yes, it was intentionally hand-crafted to cherry-pick carefully what goes in there and understand every individual one of them.

however, this addition is safe though and it does make sense (having said that, all these MLAPI words would change at some point after rebranding of product, repos & packages etc).